torch/csrc/jit/runtime/symbolic_shape_registry.cpp

```diff
@@ -58,6 +58,22 @@ const std::string shape_compute_functions =
         def unary(self: List[int]):
           return _copy(self)
 
+        def broadcast_inplace(a: List[int], b: List[int]):
```

Comment:
just curious - did you write this or find logic for it someplace ?

Reply:
I just wrote this based on just checking what broadcasts are valid for inplace. 

However, it seems based on the comment you left on test_symbolic:174 that I should not take the second argument into account at all, and instead treat all inplace ops as unary.

Reply:
No, this seems right, we want to preserve the error checking